packages/app-headless-cms/src/admin/plugins/fieldRenderers/richText/utils.ts

```diff
@@ -0,0 +1,22 @@
+import { CmsModel, CmsModelField } from "@webiny/app-headless-cms-common/types";
+
+/**
+ * Model have at least one single/multiple legacy RTE field
+ */
```

Comment:
All the comments in this file are useless, because function names are very clear. Besides not bringing any value, comments also contain grammar mistakes (`have` -> `has`), and typos (the next comment has `id` instead of `if`). Just remove all comments from this file.